it/uk/gov/hmrc/transitmovements/repositories/MovementsRepositorySpec.scala

```diff
@@ -195,23 +195,37 @@ class MovementsRepositorySpec
       )
     }
 
+  it should "return a list of movement responses for the supplied EORI if there are movements that were updated since the given time" in {
+    val dateTime = instant
+    GetMovementsSetup.setup()
+    await(repository.insert(GetMovementsSetup.departureGB3).value)
+    val result = await(repository.getMovements(GetMovementsSetup.eoriGB, MovementType.Departure, Some(dateTime)).value)
+
+    result.right.get.value should be(
+      NonEmptyList(
+        MovementWithoutMessages.fromMovement(GetMovementsSetup.departureGB2),
```

Comment:
Please create arbitrary in ModelGenerators and use that.
just like it is done for   "getMessages" should "return message responses if there are messages that were received since the given time" in {
    val dateTime = arbitrary[OffsetDateTime].sample.value

    val messages =
      NonEmptyList
        .fromList(
          List(
            arbitraryMessage.arbitrary.sample.value.copy(received = dateTime.plusMinutes(1)),
            arbitraryMessage.arbitrary.sample.value.copy(received = dateTime),
            arbitraryMessage.arbitrary.sample.value.copy(received = dateTime.minusMinutes(1))
          )
        )
        .value

Reply:
as discussed, what we are doing currently is fine